### PR TITLE
Onboard the Alpha Ids SDK

### PR DESCRIPTION
**Pull request by guilhermeluizsp**
**NOTE: This is a private SDK intended for internal use only. This SDK will not be publicly supported**

This is the onboarding of a new internal SDK used by the Azure Communication Services team. This is an important piece of software that will be used for managing Alpha IDs. Alpha ID is an alphanumeric phone number (like MICROSOFT, instead of a short code like 23451), most commonly used in the EU market, that's generally used in enterprise/high throughput scenarios when sending SMS messages.

_This is a package that is only going to be used internally._

### CODE REVIEW

File: sdk/communication/Azure.Communication.AlphaIds/tests.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: communication
+    Project: Azure.Communication.ShortCodes
```
Comment (benbp) [inline sdk/communication/Azure.Communication.AlphaIds/tests.yml:7]: I believe this is the wrong project? Should be AlphaIds.

```suggestion
    Project: Azure.Communication.AlphaIds
```

Comment (guilhermeluizsp) [reply sdk/communication/Azure.Communication.AlphaIds/tests.yml]: Good catch, my mistake. Just fixed it.

File: sdk/communication/Azure.Communication.AlphaIds/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-alpha.1 (Unreleased)
+
+### Features Added
```
Comment (danielortega-msft) [inline sdk/communication/Azure.Communication.AlphaIds/CHANGELOG.md:5]: I would suggest not leaving the changelog empty.

**Review by danielortega-msft — COMMENTED** [review]
Other than the empty changelog, it looks good to me.

**Review by danielortega-msft — APPROVED** [review]

**Pull request merged**